$NetBSD: patch-ab,v 1.4 2001/04/08 16:25:28 dmcmahill Exp $

makes the pvm demo work.  patch derived from a post to
comp.soft-sys.math.scilab

--- routines/pvm/pvm_proc_ctrl.c.orig	Fri Aug 18 07:10:38 2000
+++ routines/pvm/pvm_proc_ctrl.c	Tue Mar 27 10:18:59 2001
@@ -464,5 +464,11 @@
   }
 #else 
-  strcpy(cmd, "scilex");
+  if (path = getenv("SCI")){
+	strcpy(cmd,path);
+	strcat(cmd,"/bin/scilex");
+  }
+  else {
+  	strcpy(cmd, "scilex");
+  }
 #endif 
 #if (defined __MSC__) || (defined __ABSC__)
